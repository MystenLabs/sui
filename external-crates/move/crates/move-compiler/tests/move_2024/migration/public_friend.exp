error[E13002]: feature is deprecated in specified edition
  ┌─ tests/move_2024/migration/public_friend.move:3:5
  │
3 │     public( friend ) fun t0() {}
  │     ^^^^^^^^^^^^^^^^ 'public(friend)' is deprecated. Replace with 'public(package)'

error[E13002]: feature is deprecated in specified edition
  ┌─ tests/move_2024/migration/public_friend.move:5:5
  │
5 │     public(friend) fun t1() {}
  │     ^^^^^^^^^^^^^^ 'public(friend)' is deprecated. Replace with 'public(package)'

error[E13002]: feature is deprecated in specified edition
  ┌─ tests/move_2024/migration/public_friend.move:7:5
  │  
7 │ ╭     public(
8 │ │         friend) fun t2() {}
  │ ╰───────────────^ 'public(friend)' is deprecated. Replace with 'public(package)'

error[E13002]: feature is deprecated in specified edition
   ┌─ tests/move_2024/migration/public_friend.move:10:5
   │  
10 │ ╭     public(
11 │ │         friend
12 │ │         ) fun t3() {}
   │ ╰─────────^ 'public(friend)' is deprecated. Replace with 'public(package)'

error[E13002]: feature is deprecated in specified edition
   ┌─ tests/move_2024/migration/public_friend.move:14:5
   │  
14 │ ╭     public(
15 │ │         friend
16 │ │         /* comment */
17 │ │     ) fun t4() {}
   │ ╰─────^ 'public(friend)' is deprecated. Replace with 'public(package)'

error[E13002]: feature is deprecated in specified edition
   ┌─ tests/move_2024/migration/public_friend.move:20:5
   │  
20 │ ╭     public(
21 │ │         friend
22 │ │         /* deleted-comment */
23 │ │     )/* stays-comment */ fun t5() {}
   │ ╰─────^ 'public(friend)' is deprecated. Replace with 'public(package)'

error[E13002]: feature is deprecated in specified edition
   ┌─ tests/move_2024/migration/public_friend.move:25:14
   │
25 │     /*stays*/public(/*deleted*/friend/*deleted*/)/*stays*/fun t6() {}
   │              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 'public(friend)' is deprecated. Replace with 'public(package)'

