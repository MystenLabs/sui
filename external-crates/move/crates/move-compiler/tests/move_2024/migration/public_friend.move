module a::m {

    public( friend ) fun t0() {}

    public(friend) fun t1() {}

    public(
        friend) fun t2() {}

    public(
        friend
        ) fun t3() {}

    public(
        friend
        /* comment */
    ) fun t4() {}

    /* stays-comment */
    public(
        friend
        /* deleted-comment */
    )/* stays-comment */ fun t5() {}

    /*stays*/public(/*deleted*/friend/*deleted*/)/*stays*/fun t6() {}
}
