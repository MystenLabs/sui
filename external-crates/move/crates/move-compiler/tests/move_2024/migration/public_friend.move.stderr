[error] external-crates/move/crates/move-compiler/tests/move_2024/migration/public_friend.move: Error: tree-sitter failure on 
[error] ```
[error] {
[error] 
[error]     public( friend ) fun t0() {}
[error] 
[error]     public(friend) fun t1() {}
[error] 
[error]     public(
[error]         friend) fun t2() {}
[error] 
[error]     public(
[error]         friend
[error]         ) fun t3() {}
[error] 
[error]     public(
[error]         friend
[error]         /* comment */
[error]     ) fun t4() {}
[error] 
[error]     /* stays-comment */
[error]     public(
[error]         friend
[error]         /* deleted-comment */
[error]     )/* stays-comment */ fun t5() {}
[error] 
[error]     /*stays*/public(/*deleted*/friend/*deleted*/)/*stays*/fun t6() {}
[error] }
[error] ```
[error]     at checkErrorsCb (/Users/tedks/Projects/mysten/sui/external-crates/move/tooling/prettier-move/out/printer.js:30:23)
[error]     at Object.print (/Users/tedks/Projects/mysten/sui/external-crates/move/tooling/prettier-move/out/printer.js:39:16)
[error]     at callPluginPrintFunction (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/index.mjs:20422:20)
[error]     at mainPrintInternal (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/index.mjs:20405:18)
[error]     at mainPrint (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/index.mjs:20388:14)
[error]     at AstPath.call (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/index.mjs:19547:14)
[error]     at printModuleDefinition (/Users/tedks/Projects/mysten/sui/external-crates/move/tooling/prettier-move/out/cst/module.js:71:21)
[error]     at Object.print (/Users/tedks/Projects/mysten/sui/external-crates/move/tooling/prettier-move/out/printer.js:57:57)
[error]     at callPluginPrintFunction (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/index.mjs:20422:20)
[error]     at mainPrintInternal (file:///Users/tedks/Projects/mysten/sui/node_modules/prettier/index.mjs:20405:18)
